--- a/include/autoconf.h
+++ b/include/autoconf.h
***************
*** 224,230 ****
  /*
   * Debug Related Config
   */
! #define CONFIG_DEBUG /* DBG_871X, etc... */
  
  #ifdef CONFIG_DEBUG
  #define DBG	1	// for ODM & BTCOEX debug
--- 224,230 ----
  /*
   * Debug Related Config
   */
! // #define CONFIG_DEBUG /* DBG_871X, etc... */
  
  #ifdef CONFIG_DEBUG
  #define DBG	1	// for ODM & BTCOEX debug


